rust-tide-backend/Makefile

```diff
@@ -0,0 +1,27 @@
+build: RUSTFLAGS="-C link-arg=-fuse-ld=lld"
```

Comment:
Why is this link arg needed?
Is purpose of the `Makefile` to set the flags/env var or are there also other reasons?

Reply:
I read somewhere, that it speeds up compilation times, but I can't find the source anymore 🙈